interactions.go

```diff
@@ -74,10 +74,24 @@ type Interaction struct {
 	GuildID   string                            `json:"guild_id"`
 	ChannelID string                            `json:"channel_id"`
 	Member    *Member                           `json:"member"`
+	User      *User                             `json:"user"`
 	Token     string                            `json:"token"`
 	Version   int                               `json:"version"`
 }
 
+// IsInGuild is a utility function for checking if the interaction is sent in a guild or in a direct message
+func (i Interaction) IsInGuild() bool {
+	return i.Member != nil
+}
+
+// GetUser is a utility function for getting the User struct
```

Comment:
Maybe I don't quite good speak on English... but "getting the User struct" - wouldn't be nicer to insert there instead of `getting`, `retrieving`?